src/main/java/io/excaliburfrc/robot/subsystems/Drive.java

```diff
@@ -30,4 +38,34 @@ public Command arcadeDriveCommend(DoubleSupplier xSpeed, DoubleSupplier zRotatio
     return new RunCommand(
         () -> drive.arcadeDrive(xSpeed.getAsDouble(), zRotation.getAsDouble()), this);
   }
+
+  private Command moveToLineCommand() {
+    return arcadeDriveCommend(() -> 0.2, () -> 0)
+        .withInterrupt(leftTrigger.or(rightTrigger))
+        .andThen(
+            new SelectCommand(
+                () -> {
+                  if (rightTrigger.get())
+                    return alignCommand(rightTrigger, leftTrigger, rightLeader, leftLeader);
+                  if (leftTrigger.get())
+                    return alignCommand(leftTrigger, rightTrigger, leftLeader, rightLeader);
+                  return new PrintCommand(
+                      "ASSERTION FAILED!!!!!\nCall the programmers, something is borked");
+                }));
+  }
+
+  private Command alignCommand(
+      Trigger firstSensor,
+      Trigger secondSensor,
+      MotorController firstMotor,
+      MotorController secondMotor) {
+    return getSideReachCommand(secondMotor, secondSensor, 0.2)
+        .andThen(getSideReachCommand(firstMotor, firstSensor, -0.2));
+  }
```

Comment:
As @CohenAriel mentioned, yeah, this function is redundant: you can inline its usages (quick tip: `ctrl alt n` in IntelliJ).